core/templates/dev/head/pages/exploration-editor-page/improvements-tab/templates/answer-details-modal.template.html

```diff
@@ -0,0 +1,51 @@
+<div>
+  <div class="modal-header d-flex text-center oppia-issues-issue-dir">
+    <h3 class="modal-title">Answer details for <[learnerAnswerDetails.getStateName()]></h3>
+  </div>
+  <div class="modal-body" style="margin-left:0.5rem; margin-right: 0.5rem;">
+    <table class="table table-hover">
+      <tr ng-if="getLearnerAnswerInfos().length > 0">
+        <th scope="col">Select</th>
+        <th scope="col">Answer</th>
+        <th scope="col">Answer details</th>
+        <th scope="col">Created on</th>
+      </tr>
+      <tr ng-repeat="learnerAnswerInfo in getLearnerAnswerInfos()" class="oppia-feedback-tab-row">
+        <th scope="row">
+          <input type="checkbox" ng-change="selectLearnerAnswerInfo(learnerAnswerInfo)" ng-model="myVal">
+        </th>
+        <td ng-click="changeView(learnerAnswerInfo)">
+          <angular-html-bind html-data="getLearnerAnswerHtml(learnerAnswerInfo)">
+          </angular-html-bind>
+        </td>
+        <td ng-click="changeView(learnerAnswerInfo)">
+          <div><[getAnswerDetails(learnerAnswerInfo) | truncate:20]></div>
+        </td>
+        <td ng-click="changeView(learnerAnswerInfo)">
+          <span><[getLocaleAbbreviatedDatetimeString(learnerAnswerInfo._createdOn)]></span>
+        </td>
+      </tr>
+    </table>
+    <div class="row" style="margin-left: 0.5rem; margin-left: 0.5rem" ng-if="currentLearnerAnswerInfo">
+      <b>Answer:</b>
+      <angular-html-bind html-data="getLearnerAnswerHtml(currentLearnerAnswerInfo)">
+      </angular-html-bind>
+      <hr>
+      <b>Answer details:</b>
+      <span class="oppia-answer-details"><[getAnswerDetails(currentLearnerAnswerInfo)]></span>
+    </div>
+  </div>
+  <div class="modal-footer">
+    <button class="btn btn-danger modal-button oppia-delete-select" ng-disabled="!selectedLearnerAnswerInfo.length" ng-click="deleteSelectedLearnerAnswerInfo()">Delete selected items</button>
+    <button class="btn btn-default modal-button" ng-click="close()">Close</button>
+  </div>
+</div>
+<style>
+  .oppia-delete-select {
+    margin-right: 300px;
```

Comment:
Use a relative size instead of `px`. (eg: `vw`). Since 300px is quite big and could lead to issues in some screens.